src/routes/usuario.routes.js

```diff
@@ -0,0 +1,25 @@
+const { Router } = require('express')
+const Usuario = require('../models/usuario')
+const Token = require('../tools/token')
+
+const router = Router()
+
+router.post('/login', async(req, res) => {
+    const { email, password } = req.body
+    const usuario = await Usuario.findOne({ email })
+    if (!usuario)
+        return res.json({ msg: "No se ha encontrado el usuario" })
+            //const coincide = await usuario.compararPassword(password)
+    const coincide = password == usuario.password
+    if (!coincide)
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Backend/pullRequest?prid=7127635)
```suggestion
    if (!coincide)
```